docs/docs/getting-started/installation/installation-and-launch.md

```diff
@@ -7,29 +7,75 @@ title: Installation and launch
+be installed on any system that supports recent Docker versions. However, due to performance advantages (e.g., real-time
+task execution), AICA suggests using Linux-based platforms that can be easily configured for such considerations. 
 
-Before proceeding, [install Docker Engine](https://docs.docker.com/engine/install/) on the host machine.
+Before proceeding, you will need to install Docker Engine on the host machine. Depending on your platform, the
+installation process may vary or require additional steps. Below, you will find instructions per operating system:
 
-:::info
+<details>
+<summary>Linux</summary>
+
+While AICA currently distributes AICA Launcher for Debian-based systems, other distributions where Docker can be
+installed work equally as well through any browser. You may find distribution-specific instructions to install Docker
+Engine [here](https://docs.docker.com/engine/install/) and follow the manual installation steps to build and run AICA
+Core [here](../../reference/manual-installation-launch.md). Note that you can and it is **recommended** that you skip
+the installation of Docker Desktop for Linux.
+
+:::note
 
 For Ubuntu users, make sure to follow
-the [post installation steps](https://docs.docker.com/engine/install/linux-postinstall/) to create the `docker` group
-and add your user.
+the [post installation steps](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user)
+to create the `docker` group and add your user.
 
 :::
 
 :::caution
 
-For Ubuntu users, note that a **Docker Desktop for Linux** installation does **not** include all the necessary 
-dependencies. Follow the instructions from the provided link to ensure that all requirements are met.
+For Ubuntu users that need a **Docker Desktop for Linux**, note that its installation does **not** include all the
+necessary dependencies. Follow the instructions from the provided link to ensure that all requirements are met.
 
 When installed, Docker Desktop for Linux can **not** be used to manage images and containers created by the AICA System. 
 Additionally, you may experience some issues when using AICA System software due to Docker Desktop's custom
 configuration (see [manual installation](../../reference/manual-installation-launch.md) for more details).
 
 :::
 
+</details>
+
+<details>
+<summary>MacOS</summary>
+
+The simplest way to install Docker Engine on MacOS is through 
+[Docker Desktop](https://docs.docker.com/desktop/setup/install/mac-install/). However, some limitations may apply
+regarding commercial use (see relevant article 
+[here](https://www.docker.com/blog/revisiting-docker-hub-policies-prioritizing-developer-experience/)). If you intend to
+operate Docker Desktop within the terms of service, make sure that 
+[these](../../reference/manual-installation-launch.md#configuring-docker-on-your-linux-or-macos-system) additional configuration steps are
+taken.
```

Comment:
"If you intend to operate Docker Desktop within the terms of service" -> that's a bit of a weird sentence 😅 

```suggestion
[Docker Desktop](https://docs.docker.com/desktop/setup/install/mac-install/). Ensure that 
[these](../../reference/manual-installation-launch.md#configuring-docker-on-your-linux-or-macos-system) additional configuration steps are
taken.
Note that some limitations may apply regarding commercial use (see relevant article 
[here](https://www.docker.com/blog/revisiting-docker-hub-policies-prioritizing-developer-experience/)).
```